v4_MIGRATION_GUIDE.md

```diff
@@ -617,10 +617,12 @@ public class VenmoActivity extends AppCompatActivity {
     }
   }
 
-  private void authorizeVenmoAccount() {
-    boolean shouldVault = false; 
-    String profileId = "your-profile-id"; 
-    venmoClient.authorizeAccount(this, shouldVault, profileId, (error) -> {
+  private void tokenizeVenmoAccount() {
+    VenmoRequest request = new VenmoRequest()
+            .profileId("your-profile-id")
+            .shouldVault(false);
+          
+    venmoClient.tokenizeVenmoAccount(this, request, (error) -> {
```

Comment:
I think it wouldn't hurt to add a sentence before the code snippet calling out that this method was renamed. Maybe something like:

```
The `authorizeVenmoAccount` method has been replaced with`tokenizeVenmoAccount`. 
```

Reply:
Good idea, I added a note to this method in the migration guide.